### DISCUSSION TITLE
Issue with Nested resources in resourceTimelineDay

### DISCUSSION THREAD

**Discussion by nrdnisml** [GuavaCZ/calendar#55] [Q&A] [answered]
Facing an issue with the resourceTimelineDay. According to the documentation [vkurko Calendar](https://github.com/vkurko/calendar#resource-object) it should be possible to add child resources using the children key in the resource object.

Here is my code:
```
public function getResources(): Collection|array
    {
        return [
            [
                'id' => 'a',
                'title' => 'floor 1',
                'children' => [
                    ['id' => '1', 'title' => 'Room 1'],
                    ['id' => '2', 'title' => 'Room 2'],
                ]
            ],
            [
                'id' => 'b',
                'title' => 'floor 2',
                'children' => [
                    ['id' => '3', 'title' => 'Room 3'],
                    ['id' => '4', 'title' => 'Room 4'],
                ]
            ],
        ];
    }
```

However, the result appears as follows:
![image](https://github.com/user-attachments/assets/fc726750-245b-48ea-98b1-7150a04f089c)

It seems like the children key is not being executed properly, and the child resources are not displayed as expected.

How can I resolve this issue? Am I missing something in the implementation?

**Comment by lukas-frey** [comment]
Nested resources is new in vkurko/calendar and this package hasn't been updated to it yet.

**Comment by lukas-frey** [comment, marked as answer]
Version 1.11.0 now supports nested resources in resource timeline views.
